providers/utils.go

```diff
@@ -71,6 +71,10 @@ func ReadCatalog(opts ...CatalogOption) (CatalogType, error) {
 // provider information. As a general rule, don't ever call this function
 // in production code unless you have a compelling reason to do so.
 func SetInfo(provider Provider, info ProviderInfo) {
```

Comment:
The comments above this function are now outdated - since you are now definitely calling this function in production code.

Reply:
Lol fair. Yes I backtracked on that pretty quickly. I'll update the comments.